packages/trpc/server/createNextApiHandler.ts

```diff
@@ -64,6 +64,7 @@ export function createNextApiHandler(router: AnyRouter, isPublic = false) {
           // Revalidation time here should be 1 second, per https://github.com/calcom/cal.com/pull/6823#issuecomment-1423215321
           "slots.getSchedule": `no-cache`, // FIXME
           cityTimezones: `max-age=${ONE_DAY_IN_SECONDS}, stale-while-revalidate`,
+          map: `max-age=${ONE_DAY_IN_SECONDS}, stale-while-revalidate`, // "map" - Feature Flag Map
```

Comment:
This approach of mapping cacheRules can give unexpected results as it isn't checking for full path. So, there can be features.map and example.map. This cache header would be applied to both I guess?

This isn't related to the PR, so it's non blocking but should be fixed I guess.

Reply:
Oh i guess youre right -> We could rename this route to something more specific so this doesnt happen?

Reply:
Also, this is going to consider the response fresh for a day. We might not be able to disable/enable a feature for our users within a good amount of time which is an important thing IMHO.

Also, I suspect a weird issue with this
We might end up in a scenario like this
- Say, a feature F1 was disabled and thus it's cached across all our users.
- We disable Feature F1, now, backend code knows immediately that F1 is disabled but frontend won't realize it and it might keep showing F1 in UI and taking action on that feature would be rejected by server(as it knows feature is disabled).

Reply:
I think the approach to solve the multiple requests for flag might be to configure react-query in such a way that the request goes only once from one pageview. Only, on hard refresh we send a new request.

Above scenario can still happen in here, but user can always refresh the page

Reply:
Using stale while revalidate will prevent stale data for more than a few seconds

Reply:
Pushing a fix for the namespace issue

<img width="773" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/f57b4fa3-5c9c-49c0-b6bf-aa221c8e4732">